### PR TITLE
add last event command

### PR DESCRIPTION
**Pull request by vdbg**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -318,29 +320,31 @@ def command_timezone(self, update: Update, context: CallbackContext) -> None:
             else:
                 self.send_text(update, context, "No timezone set for current user. Using UTC.")
 
+    def command_device_last_event(self, update: Update, context: CallbackContext) -> None:
+        self.get_device_events(update, context, True)
+
     def command_device_events(self, update: Update, context: CallbackContext) -> None:
+        self.get_device_events(update, context, False)
+
+    def get_device_events(self, update: Update, context: CallbackContext, is_last: bool) -> None:
```
Comment (Nickolaim) [inline main.py:329]: maybe rename `is_last` to `last_only`?

File: main.py [hunk 2]
```diff
@@ -511,11 +529,12 @@ def configure(self) -> None:
         self.add_command(["close"], "close device `name`", self.command_device_close, AccessLevel.DEVICE, params="name")
         self.add_command(["commands", "c"], "list supported commands for device `name`", self.command_device_commands, AccessLevel.DEVICE, params="name")
         self.add_command(["dim", "d"], "dim device `name` by `number` percent", self.command_device_dim, AccessLevel.DEVICE, params="number name")
-        self.add_command(["events", "e"], "get recent events for device `name`", self.command_device_events, AccessLevel.SECURITY, params="name")
+        self.add_command(["events", "e"], "get last 20 events for device `name`", self.command_device_events, AccessLevel.SECURITY, params="name")
```
Comment (Nickolaim) [inline main.py:532]: Not sure where 20 is coming from.  The code is calling `self.hubitat.api.get_device_events(device.id)` and then iterating over them.  If Hubitat returns 20 events exactly and is never going to change this number than I guess the code is fine.  I like `recent` events more - less specific and still correct.

**Review by Nickolaim — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (vdbg) [reply main.py:329]: ok

File: main.py (hunk 2 above)
Comment (vdbg) [reply main.py:532]: ok

**Pull request merged**